releasenotes/notes/remove-circuit-mcmt-87fe50eb32006f7f.yaml

```diff
@@ -0,0 +1,22 @@
+upgrade:
+  - |
+    Removed the `QuantumCircuit.mcmt` method which has been deprecated with the 
+    Qiskit Terra 0.14.0 release in April 2020. Instead of using the method,
+    please use the :class:`~qiskit.circuit.library.MCMT` class instead to construct
+    a multi-control multi-target gate and use `QuantumCircuit.append` or 
+    `QuantumCircuit.compose` to add it to a circuit. 
+
+    For example, you can replace
+    
+        circuit.mcmt(ZGate(), [0, 1, 2], [3, 4])
+
+    with
```

Comment:
```suggestion
    with::
```

Reply:
Also can you add the import statement here because it won't be obvious from the rendered doc that it'd be:
`from qiskit.circuit.library import MCMT`

Reply:
Good point, thanks!